docs/assets/templates/workflows/barebones-multiple.dockstore.yml

```diff
@@ -0,0 +1,22 @@
+version: 1.2
```

Comment:
retitle "recommended-multiple-workflows-per-repository.dockstore.yml" and use this more descriptive language in other examples.

Reply:
I did give that idea a go, but it made things hard to organize for maintainers of the documentation. Went with a number example scheme instead. Let me know what you think of it.